recipes-debian/net-tools/net-tools_debian.bb

```diff
@@ -67,15 +70,13 @@ base_sbindir_progs = "ipmaddr iptunnel mii-tool nameif"
 base_bindir_progs  = "ifconfig route netstat"
 # Packages disabled in config file. Add to enable.
 # base_sbindir_progs += " arp plipconfig rarp slattach"
-# base_bindir_progs  += " dnsdomainname domainname hostname nisdomainname ypdomainname"
+base_bindir_progs  += " dnsdomainname domainname hostname nisdomainname ypdomainname"
```

Comment:
Original recipe in OE(Poky) locates programs in different manner.
```
base_sbindir_progs = "arp ifconfig ipmaddr iptunnel mii-tool nameif plipconfig rarp route slattach"
base_bindir_progs  = "dnsdomainname domainname hostname netstat nisdomainname ypdomainname"
```
Can you follow this? If you want to focus on hostname, you can ignore other programs(e.g. arp) now and please leave that in comments.

Reply:
OK. Done.